<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>babelGUIClass</class>
 <widget class="QMainWindow" name="babelGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>babelGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QLabel" name="usernameLbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StatusBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#StatusBtn{
 	border:none;
	background:none;
	image: url(:/babelGUI/Resources/Online.png);
}

QPushButton#StatusBtn:hover{
 	border:none;
	background:none;
	image: url(:/babelGUI/Resources/change.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="homeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#homeBtn{
	border:none;
	background:none;
	image: url(:/babelGUI/Resources/home.png);
}

QPushButton#homeBtn:hover{
 	border: 1px;
    border-radius: 15px;
	background-color:rgba(240, 177, 8, 0.3);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="babelgui.qrc">:/babelGUI/Resources/lilrom.jpg</pixmap>
    </property>
   </widget>
   <widget class="SlidingStackedWidget" name="stakedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>131</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:rgb(240, 177, 8);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Contacts</string>
     </attribute>
     <widget class="QListWidget" name="friendList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color:rgb(240, 177, 8);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Actions</string>
     </attribute>
     <widget class="QListWidget" name="actionList">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>141</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <zorder>photo</zorder>
   <zorder>usernameLbl</zorder>
   <zorder>StatusBtn</zorder>
   <zorder>homeBtn</zorder>
   <zorder>tabWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>stakedWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="babelgui.qrc"/>
 </resources>
 <connections/>
</ui>
